bpfd-operator/config/rbac/bpfprogramconfig_editor_role.yaml

```diff
@@ -9,7 +9,7 @@ metadata:
     app.kubernetes.io/created-by: bpfd-operator
     app.kubernetes.io/part-of: bpfd-operator
     app.kubernetes.io/managed-by: kustomize
-  name: BpfProgramConfig-editor-role
+  name: bpfprogramconfig-editor-role
```

Comment:
Looking through the directors for these files, I stumbled on `bpfd-operator/config/crd/patches/` which has:
```
cainjection_in_ebpfprogramconfigs.yaml
cainjection_in_ebpfprograms.yaml
webhook_in_ebpfprogramconfigs.yaml
webhook_in_ebpfprograms.yaml
```
which should probably also be renamed.

Reply:
++ ack I'll do that

Reply:
Done